tests/python/gaia-ui-tests/gaiatest/apps/clock/regions/new_alarm.py

```diff
@@ -37,7 +37,11 @@ def alarm_label_placeholder(self):
 
     @property
     def alarm_repeat(self):
-        return self.marionette.find_element(*self._repeat_menu_locator).text
+        repeat = self.marionette.find_element(*self._repeat_menu_locator).get_attribute('data-l10n-id')
+        if repeat is None:
```

Comment:
This can be shortened by a ternary operator:
``` py
return repeat if repeat is not None else self.marionette.find_element(*self._repeat_menu_locator).text
```